.release-please-manifest.json

```diff
@@ -1,11 +1,12 @@
 {
-  "packages/app-info": "1.2.1",
-  "packages/crash-handler": "1.1.4",
-  "packages/errors": "1.3.0",
-  "packages/log-error": "1.5.4",
-  "packages/logger": "1.1.2",
-  "packages/middleware-log-errors": "1.5.4",
-  "packages/middleware-render-error-info": "1.1.17",
-  "packages/serialize-error": "1.1.4",
-  "packages/serialize-request": "1.1.1"
```

Comment:
Package to add back in: "packages/eslint-config": "0.0.0"

Reply:
Somehow my fix has resulted in more white spaces 😭  but my file on VS Code doesn't look affected

Reply:
Ah this is one of those rare cases where we use two-spaces for indentation 🙂 it's because Release Please automatically updates this file and resets the indentation to two-spaces each time, it's not what we'd generally use but Release Please doesn't give us the option

Reply:
Ok I'll try and delete the extra space and see if that fixes it - thanks! 👍🏽

Reply:
Fixed by manually deleting down the indents from 4 to 2! Thank you 😁